kore/src/main/scala/org/kframework/definition/outer.scala

```diff
@@ -97,6 +97,18 @@ case class Module(val name: String, val imports: Set[Module], localSentences: Se
       .groupBy(_.sort)
       .map { case (l, ps) => (l, ps) }
 
+  lazy val layouts: Set[String] =
+    productionsForSort
+      .get(ADT.Sort("Layout"))
+      .getOrElse(Set[Production]())
+      .collect({
+          case Production(_, Seq(Terminal(terminalString)), _)           => terminalString
```

Comment:
You need to escape special characters here.

Reply:
Well, I added print statements in `Scanner.java` to make sure the correct strings were making it through given the above `DEFAULT-LAYOUT` module. Basically, I left in the old `singleLine` and `multiLine` constants, and printed them side-by-side with the generated `layout`, and saw that they were the exact same. Perhaps string handling has already happened by the time it's dumped into `Terminal` and `RegexTerminal` by the parser?

Reply:
Ah, I see what you mean. Hmmmm, I'll have to look and see if there is a way to interpret an entire string as literals instead of a regex.